src/Resources/Resources.Test/ScenarioTests/DeploymentTests.ps1

```diff
@@ -81,7 +111,7 @@ function Test-CrossResourceGroupDeploymentFromTemplateFile
 	$rgname = Get-ResourceGroupName
 	$rgname2 = Get-ResourceGroupName
 	$rname = Get-ResourceName
-	$rglocation = "CentralUSEUAP"
+	$rglocation = "West US 2"
```

Comment:
Can we fix this?  If not, can we file an issue to fix this?

Reply:
@cormacpayne added comment about fixing these hard-coded locations across ARM tests in the issue that was filed for the other Resources PR: https://github.com/Azure/azure-powershell/issues/8678